### PR TITLE
Add assignees to update merge request request

### PR DESCRIPTION
**Pull request by TalfinApiiro**
I despise the original maintainer of this library, half assing it entirely.
We need assignees for the workflow.

### CODE REVIEW

**Comment by barapiiro** [comment]
@liiri is it going to be published automagically or need to publish it manually?

**Review by barapiiro — APPROVED** [review]

**Comment by TalfinApiiro** [comment]
> @liiri is it going to be published automagically or need to publish it manually?

Manually, as described in the readme

**Comment by barapiiro** [comment]
> > @liiri is it going to be published automagically or need to publish it manually?
> 
> Manually, as described in the readme

👍🏽 
I was lazy and the readme are not in the PR :)

File: src/GitLabApiClient/Models/MergeRequests/Requests/UpdateMergeRequest.cs [hunk 1]
```diff
@@ -33,6 +33,12 @@ public sealed class UpdateMergeRequest
         [JsonProperty("assignee_id")]
         public int? AssigneeId { get; set; }
 
+        /// <summary>
+        /// Assignees user IDs.
+        /// </summary>
+        [JsonProperty("assignee_ids")]
+        public int? AssigneeIds { get; set; }
```
Comment (liiri) [inline src/GitLabApiClient/Models/MergeRequests/Requests/UpdateMergeRequest.cs:40]: how come "ids" is a single int ?

**Pull request merged**